test/performance/README.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+
+The purpose of this Kubernetes CNI scale testing tool is to evaluate the performance and scalability of
+CNI within Kubernetes clusters.
+
+The following sections detail these features and explain how to use the tool.
+
+## 1. Scalability
+
+Whether it's a cluster with a single node or hundreds of nodes, you can execute tests with just
+a command line.
+
+```shell
+make bin
+./bin/antrea-scale --kubeConfigPath=/root/.kube/config --timeout=120 --config=./test/performance/scale.yml
```

Comment:
You can print the help message of this command `antrea-scale` before introduce the detail usages, otherwise, it's hard for user to understand what's timeout unit.
```
Usage of ./bin/antrea-scale:
      --config string              Config file of scale test cases list (default "test/performance/scale.yml")
      --kubeConfigPath string      Path of cluster kubeconfig
      --templateFilesPath string   Template YAML files path of test cases (default "test/performance/assets")
      --timeout int                Timeout limit (minutes) of the whole scale test (default 10)
      --v string                    (default "2")
```